java/README.md

```diff
@@ -0,0 +1,22 @@
+## Java SDK samples
```

Comment:
Add this information to the general readme.
We want to be displayed on this page: https://github.com/RWS/languageweaver-sdk-samples/.